omnibus/package-scripts/agent-deb/postinst

```diff
@@ -102,11 +102,15 @@ chown -R dd-agent:dd-agent ${CONFIG_DIR}
 chown -R dd-agent:dd-agent ${LOG_DIR}
 chown -R dd-agent:dd-agent ${INSTALL_DIR}
 
+
+# Ensure system-probe config exists
+if [ ! -f "$CONFIG_DIR/system-probe.yaml" ]; then
+    cp "$CONFIG_DIR/system-probe.yaml.example" "$CONFIG_DIR/system-probe.yaml"
+fi
+
 # Make system-probe configs read-only
 chmod 0440 ${CONFIG_DIR}/system-probe.yaml.example || true
-if [ -f "$CONFIG_DIR/system-probe.yaml" ]; then
-    chmod 0440 ${CONFIG_DIR}/system-probe.yaml || true
-fi
+chmod 0440 ${CONFIG_DIR}/system-probe.yaml || true
```

Comment:
All I know is that our documentation tells users to create the file with `0640` permissions, and you will need write permissions to make any changes. The revert PR does not specify the reasons but I doubt they had to do with the system-probe.yaml file because there were a lot of other changes.